articles/sentinel/normalization.md

```diff
@@ -107,6 +107,43 @@ Each schema field has a type. Some have built-in, Azure Log Analytics types such
 
 <br>
 
+### Common fields
+
+The following fields are common to all ASIM schemas. They are documented here, but also for each schema, as there might be unique details relevant to each schema. For example the valid values for EventType or the valud of EventSchemaVersion. 
+
+| Field               | Class       | Type       |  Description        |
+|---------------------|-------------|------------|--------------------|
+| <a name="timegenerated"></a>**TimeGenerated** | Built-in | datetime | The time the event was generated by the reporting device.|
+| **_ResourceId**   | Built-in |  guid     | The Azure Resource ID of the reporting device or service, or the log forwarder resource ID for events forwarded using Syslog, CEF or WEF. |
```

Comment:
@oshezaf missing oxford comma:

Syslog, CEF, or WEF.

Reply:
Now I know how it is called :-)